habitat-hitl/habitat_hitl/core/client_message_manager.py

```diff
@@ -73,3 +73,21 @@ def update_navmesh_triangles(self, triangle_vertices):
         self._message["navmeshVertices"] = [
             component for sublist in triangle_vertices for component in sublist
         ]
+
+    def update_camera_transform(
+        self, pos: List[float], rot_quat: List[float]
```

Comment:
ClientMessageManager should internally do the type conversion required for serialization. More intuitive arguments here would be Magnum Vector3 and Quaternion.